subprojects/docs/src/docs/userguide/gradleCommandLine.adoc

```diff
@@ -48,6 +48,14 @@ Set to `auto` (the default) to enable color and other rich output in the console
 Set to `rich` to enable color and other rich output in the console output, regardless of whether the build process is not attached to a console. When not attached to a console, the build output will use ANSI control characters to generate the rich output.
 +
 Set to `verbose` to enable color and other rich output like the `rich`, but output more detailed message (task name, output, etc.)
+`--warnings`::
+Specifies which type of deprecation warnings displayed in console.
```

Comment:
❌ We should explicitly call them "Gradle deprecation warnings".  This setting does not affect compiler warnings and warnings from other tools.

Reply:
Personally I don't like to introduce too many build options. So here I think we can use `--warnings=no-deprecation` instead of an extra `--dep-warnings=none`. This allow us to extend the "warning type" in the future.

Reply:
I wasn't talking about the name of the option, but its description. The description should clearly say these are *Gradle*  warnings, not Java compiler warnings etc. Unless we want to unify all of those.